example.lisp

```diff
@@ -47,15 +47,15 @@
 (def-view last-name :lessp (person :test-graph)
   (:map
    (lambda (person)
-     (when (last-name person)
-       (yield (last-name person) nil)))))
```

Comment:
I am not sure why you dropped the use of the convenience methods that are defined as part of the macro expansion of `def-vertex` and `def-edge`, in this case, `last-name`.

Reply:
The accessor method is not recorgnized by LispWorks. I have not checked the reason yet but it is.